### DISCUSSION TITLE
UI - Find Releases window - previously tagged release ID field

### DISCUSSION THREAD

**Discussion by alectron** [ghDaYuYu/foo_discogs#102] [Q&A] [answered]
Hi,
this feels like it should be as it was before - but I only noticed now, no idea how long its' been like this, and this could be intentional or due to technical limitations as well:
Historically, with previously tagged files [that have the DISCOGS_RELEASE_ID field set in metadata] the UI - Find Releases window - the release ID field (bottom right entry field) seems to have gotten the discogs  ID value directly from the file metadata and it was filled immediately.
On the current release, [1.0.17.4](https://github.com/ghDaYuYu/foo_discogger/releases/tag/v1.0.17.4), for me this only is filled with the string when I select the (master) release in the same windows' 'releases' list, which feels odd (as it looks as if it hasn't been tagged before).

Hm, ps: actually turns out - looks like the wiring underneath has changed and the problem/change is rather different: 
The olde field
DISCOGS_RELEASE_ID
is greyed out in the "tag map..." UI and was set to disabled for me for some unknown reason. So, with the info here;
https://hydrogenaud.io/index.php/topic,50523.msg1027458.html#msg1027458
I set it to write + update with the shift trick, and now it's writing the ID again, as well as displaying it as intended.

I'll post just in case anyone else is coming across this and/or it helps anyone else.
Cheers.

**Comment by ghuDaYuYu** [comment, marked as answer]
Hi alectron,

First of all thanks for taking the time to post feedback as well as a troubleshooting reference for people who come across the same issue.

The release ID field was empty because its metadata was missing then, as I understand it, due to the DISCOGS_RELEASE_ID mapping being disabled for some mysterious reason.

Mentioning the historical context, a new feature has been added recently after repeated requests to enable all map action options related to the DISCOGS_RELEASE_ID field, in particular the 'Disable' option.

As a result, there are now two modes to operate the component.

- The original 'Write Tags...' mode which now also includes the ability to choose 'Write and update' action for the release ID field using the shift + right click trick.
- The new mode 'Plain write tags...' that also enables the 'Disable' DISCOGS_RELEASE_ID map action.

Both modes share the same tag mapping definitions and in theory, the component should remember the last mode's DISCOGS_RELEASE_ID setting, restoring it accordingly next time each mode becomes active again.

And it seams there might be a bug in this late feature, causing the wrong map action ('Disable') being restored instead of 'Write' or 'Write and update'.

Either that or maybe the 'Plain write tags...' was inadvertently chosen.

It might have been a bad decision to introduce this new mode, still considering the pros and cons of having done so; some people may find some usefulness in the 'plain' mode not forcibly relying on TXXX frames, but it also adds complexity, maybe new bugs... time will tell.

Sorry for long text with unrelated details, glad you found a solution to this issue.
Regards
